packages/react-components/react-search/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -109,14 +104,20 @@ export const useSearchBoxStyles_unstable = (state: SearchBoxState): SearchBoxSta
       dismissClassName,
       disabled && dismissStyles.disabled,
       dismissStyles[size],
+
+      !focused && contentAfterStyles.unfocused,
```

Comment:
I think you only need this on `contentAfter`, since `dismiss` is nested within it, and can't exist without the `contentAfter` slot